docs/settings.md

```diff
@@ -44,3 +44,50 @@
+| `session-ping-limit`                 | int                                                              | `200`      |        | Number of sessions the HPB can ping in a single request                                                                                                                          |
+| `grid_videos_limit`                  | int                                                              | `19`       |        | Maximum number of videos to show (additional to the own video)                                                                                                                   |
+| `token_entropy`                      | int                                                              | `8`        |        | Length of conversation tokens, can be increased to make tokens harder to guess but reduces readability and dial-in comfort                                                       |
+| `default_group_notification`         | int                                                              | `2`        | 🖌️    | Default notification level for group conversations [constants list](constants.md#participant-notification-levels)                                                                |
+| `conversations_files`                | string<br>`1` or `0`                                             | `1`        | 🖌️    | Whether the files app integration is enabled allowing to start conversations in the right sidebar                                                                                |
+| `conversations_files_public_shares`  | string<br>`1` or `0`                                             | `1`        | 🖌️    | Whether the public share integration is enabled allowing to start conversations in the right sidebar on the public share page (Requires `conversations_files` also to be enabled |
+| `enable_matterbridge`                | string<br>`1` or `0`                                             | `0`        | 🖌️    | Whether the matterbridge integration is enabled and can be configured                                                                                                            |
+| `changelog`                          | string<br>`yes` or `no`                                          | `yes`      |        | Whether the changelog conversation is updated with new features on major releases                                                                                                |
+| `grid_videos_limit_enforced`         | string<br>`yes` or `no`                                          | `no`       |        | Whether the number of grid videos should be enforced                                                                                                                             |
```

Comment:
Are the properties in a specific order? If not it might be better to move `grid_videos_limit_enforced` just below `grid_videos_limit`.

Reply:
I can rearrange the segments, but I grouped:
arrays > strings > int > boolean with 1/0 > boolean with yes/no

But we can swap the last two and then add it first/last